src/nrc/api/viewsets.py

```diff
@@ -95,29 +83,25 @@ class KanaalViewSet(
     }
 
 
+@extend_schema(summary="Publiceer een notificatie.")
 class NotificatieAPIView(views.APIView):
     """
     Publiceren van NOTIFICATIEs.
 
     Een NOTIFICATIE wordt gepubliceerd op een KANAAL. Alle consumers die een
     ABONNEMENT hebben op dit KANAAL ontvangen de NOTIFICATIE.
-
-    create:
-    Publiceer een notificatie.
     """
 
     required_scopes = {"create": SCOPE_NOTIFICATIES_PUBLICEREN}
     # Exposed action of the view used by the vng_api_common
     action = "create"
+    serializer_class = MessageSerializer
 
-    @swagger_auto_schema(
-        request_body=MessageSerializer, responses={200: MessageSerializer}
-    )
```

Comment:
This does mean that the status code 200 response is no longer in the API spec, just to double check but was this a mistake in the previous API spec and do we return a 201 instead?